Woof/Woof/View/OwnerCardView/OwnerCardView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+/// A view displaying information about the pet owner.
+struct OwnerCardView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: OwnerCardViewModel
+
+    var body: some View {
+        HStack {
+            AvatarView(url: viewModel.imageURL)
+
+            VStack(alignment: .leading, spacing: 8) {
+                Text(viewModel.fullName)
+                    .font(
+                        .system(
+                            size: AppStyle.FontStyle.heading.size, weight: .heavy
+                        ))
+                    .lineLimit(1)
```

Comment:
Why do the name and surname combine?
They are placed on separate lines in mocks.

Reply:
Fixed it.